docs/sequence/PISPLinkingApi.puml

```diff
@@ -0,0 +1,577 @@
+end note
+PISP -> PISP_TP_OUT: POST /linking/request-consent
+rnote right of PISP #LightBlue
+POST /linking/request-consent
+{
+  "accounts": [
+    { accountNickname: "XXXXXXnt", id: "dfspa.username.1234", currency: "ZAR" },
+    { accountNickname: "SpeXXXXXXXXnt", id: "dfspa.username.5678", currency: "USD" }
+  ],
+  callbackURI: 'pisp-app://callback'
+}
+end note
+
+
+activate PISP
+activate PISP_TP_OUT
+
+PISP_TP_OUT -> PISP_TP_OUT: const model = await create()
+rnote right of PISP_TP_OUT: state: start
+PISP_TP_OUT -> PISP_LM: model.requestConsent()
+
+activate PISP_LM
+
+PISP_LM -> PISP_LM: ThirdpartyRequests.postConsentRequests()
+PISP_LM -> Switch: POST /consentRequests
+rnote right of PISP_LM #LightBlue
+POST /consentRequests
+FSIOP-Source: pispa
+FSIOP-Destination: dfspa
+{
+  // consentRequestId will be the uuid created
+  // at the PRELINKING-2
+  consentRequestId: 6789
+  scopes: [{
+    accountId: 'dfspa.username.1234',
+    actions: ['accounts.getBalance', 'accounts.transfer'],
+    accountId: 'dfspa.username.5678',
+    actions: ['accounts.getBalance', 'accounts.transfer'],
+  }],
+  // model will add `authChannels`
+  authChannels: ["WEB", "OTP"],
+  callbackURI: 'pisp-app://callback...'
+}
+end note
+
+activate Switch
+Switch --> PISP_LM: 202 Accepted
+deactivate PISP_LM
+Switch -> DFSP_TP_IN: POST /consentRequests
+activate DFSP_TP_IN
+
+DFSP_TP_IN -> DFSP_TP_IN: const model = await create()
+activate DFSP_LM
+rnote right of DFSP_TP_IN: state: start
+DFSP_TP_IN -> DFSP_LM: model.requestConsent()
+DFSP_TP_IN --> Switch: 202 Accepted
+deactivate Switch
+deactivate DFSP_TP_IN
+DFSP_LM -> DFSP_LM: DFSPBackendRequests.validateConsentRequest()
+activate DFSPAuthorizeSimulator
+DFSP_LM -> DFSPAuthorizeSimulator: ""POST /store/consentRequests/6789""
+DFSPAuthorizeSimulator -> DFSPAuthorizeSimulator: store consentRequest details
+DFSPAuthorizeSimulator -> DFSP_LM: 201 Created
+rnote right of DFSP_LM: state: consentRequestValidatedAndStored
+deactivate DFSPAuthorizeSimulator
+
+activate Switch
+DFSP_LM -> Switch: PUT /consentRequests/6789
+
+rnote left of DFSP_LM #LightBlue
+PUT /consentRequests/6789
+FSIOP-Source: pispa
+FSIOP-Destination: dfspa
+{
+  consentRequestId: 6789
+  scopes: [{
+    accountId: 'dfspa.username.1234',
+    actions: ['accounts.getBalance', 'accounts.transfer'],
+    accountId: 'dfspa.username.5678',
+    actions: ['accounts.getBalance', 'accounts.transfer'],
+  }],
+  authChannels: ["WEB"],
+  callbackURI: 'pisp-app://callback...',
+  authURI: 'dfspa.com/authorize?consentRequestId=6789' // this is new
+}
+end note
+Switch --> DFSP_LM: 202 ACCEPTED
+deactivate DFSP_LM
+activate PISP_TP_IN
+Switch ->  PISP_TP_IN: PUT /consentRequests/6789
+PISP_TP_IN --> Switch: 200 OK
+deactivate Switch
+activate PISP_LM
+PISP_TP_IN --> PISP_LM: Consent Request response recieved
+rnote right of PISP_LM: state: webAuthenticationChannelResponseRecieved
+PISP_LM --> PISP_LM: this.saveToKVS()
+deactivate PISP_TP_IN
```

Comment:
move to L120, just before `rnote right of PISP_LM: state: webAuthenticationChannelResponseRecieved`